### ISSUE TITLE
Tags :: make tags clickable in popup

### ISSUE THREAD

**Issue by nickkapur** [Japan-Digital-Archives/Japan-Digital-Archive#577]

**Comment by nickkapur** [comment]
@corinnecurcie I see that you've done this, and it seems to be working fine. But do you think that when a tag is clicked it should automatically close out of the popup? That seems more intuitive...

**Comment by corinnecurcie** [comment]
Sorry, I'm not sure exactly what you mean. Isn't clicking the tag supposed
close the popup to bring up a search for the tag? That's what it's doing on
my end.

On Thu, Sep 12, 2013 at 6:00 PM, nickkapur notifications@github.com wrote:

> @corinnecurcie https://github.com/corinnecurcie I see that you've done
> this, and it seems to be working fine. But do you think that when a tag is
> clicked it should automatically close out of the popup? That seems more
> intuitive...
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/Japan-Digital-Archives/Japan-Digital-Archive/issues/577#issuecomment-24359480
> .

## 

Corinne Curcie
Harvard University
Class of 2015
Computer Science

**Comment by nickkapur** [comment]
@corinnecurcie Thanks for the reply. Okay so I've tested this out in different browsers and machines. In both Chrome and Firefox on the Windows 7 machine in my office, and in Chrome on my Macbook, when I click on the tag I can see that it pulls up the correct search, but the popup is not closed out. Instead the popup remains until I close out of it manually. The only time the popup closes out is in Firefox on the Macbook. Weird.

**Comment by nickkapur** [comment]
Come to think, maybe I need to clear out the cache? I'm pretty sure I did that, but let me try again and get back to you.

**Comment by nickkapur** [comment]
Okay, nevermind, it works!  Yet another object lesson for me about the need to clear the cache every. single. time. Sorry about the confusion - my bad.
